### ISSUE TITLE
Start container in pwd

### ISSUE THREAD

**Issue by nitrocode** [cloudposse/geodesic#726]
Have a question? Please checkout our [Slack Community](https://slack.cloudposse.com) or visit our [Slack Archive](https://archive.sweetops.com/). 

[![Slack Community](https://slack.cloudposse.com/badge.svg)](https://slack.cloudposse.com)

## Describe the Feature

Start geodesic in the pwd where the container was run from

## Use Case

I always cd into /localhost until i get to my pwd which isnt a big deal to do but it takes time and id prefer if the container automatically did this for me without updating preferences.

## Describe Ideal Solution

pwd

## Alternatives Considered

Maintaining a long list of aliases that cd into common directories

## Additional Context
N/A

**Comment by Nuru** [comment]
@nitrocode Part of the point of Geodesic is to provide a consistent environment so multiple engineers can use the exact same tools and workflows and avoid the "works in my environment" trap. That is why the default working directory is inside the container. 

That said, we have made Geodesic customizable, so you can do what you want this way (among other options):
```
$ GEODESIC_DOCKER_EXTRA_ARGS="--env LOCAL_PWD=$PWD" make run
...

 ⧉  Geodesic 
 ✗ . [none] / ⨠ cd $LOCAL_PWD 
```

And of course you can automatically execute that command, too. For example, add to `$HOME/.geodesic/defaults/overrides` 
```
[[ -n $LOCAL_PWD ]] && cd $LOCAL_PWD 
```

**Comment by nitrocode** [comment]
Re-opening this ticket since @osterman said last Friday during the architecture meeting that he wants to see this implemented.

**Comment by Nuru** [comment]
Implemented in #756
